.github/workflows/finishReleaseCycle.yml

```
@@ -80,6 +80,21 @@ jobs:
     with:
       TARGET_BRANCH: production
 
+  # Remove DeployBlockerCash label from issues that had it since we just deployed production
+  cleanupDeployBlockers:
+    needs: updateProduction
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          gh issue list \
+            --state all \
+            --label DeployBlockerCash \
+            --json number \
+            --jq '.[].number' \
+            | xargs -L1 --max-procs=4 gh issue edit "$1" --remove-label DeployBlockerCash
+        with:
+          GITHUB_TOKEN: ${{ github.token }}
```

Does this need to be 
```suggestion
        env:
          GITHUB_TOKEN: ${{ steps.setupGitForOSBotify.outputs.OS_BOTIFY_API_TOKEN }}

```
An action is failing here.

---

ah fuck it needs to be `env` not `with`. But the normal `github.token` I think should have permissions enough to fiddle with labels. It's also a GitHub App with a higher rate limit, so no need to setup OSBotify in this case